README.md

```diff
@@ -25,3 +25,7 @@ The code can be found in `src/dashboard`.
 These three apps together form an imaginary, yet realistic, factory solution on the Actyx platform.
 
 ![factory solution](./images/advanced-tutorial-nodes.svg)
+
+### What are those weird start/end comments?
```

Comment:
They are not in the advanced example (only in the academy project), so this section can be removed.

Reply:
Removing